feature/system/aaa/README.md

```diff
@@ -0,0 +1,112 @@
+# SYS-3.1: AAA and TACACS+ Configuration Verification Test Suite
+
+
+## Summary
+
+This test suite aims to thoroughly validate the correct implementation  of the AAA (Authentication, Authorization, and Accounting) framework with TACACS+ and local authentication on a device. 
+
+## Testbed type
+
+*  [`featureprofiles/topologies/dut.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/dut.testbed)
+
+## Procedure
+
+### Test environment setup
+
+#### Configuration
+
+*   Configure the loopback interface with IPv4 and IPv6 address and netmasks of /32, /64 respectively
+
+### SYS-3.1.1 User Configuration Test
+
+*   Create a user on the DUT with the following parameters:
+    *   username: testuser
+    *   password: password
+    *   password-type: MD5
+    *   role: admin
+*   Verification:
+    *   Confirm the successful creation of the "testuser" account with the specified parameters.
+    *   Validate that the password is encrypted using MD5.
+
+### SYS-3.1.2 TACACS+ Server Configuration Test
+
+*   Configuration:   
+    *   Configure a TACACS server on the DUT as below:
+        *   Host IP: 192.168.1.1
+        *   Port: 49
+        *   Key: tacacs_password
+        *   Timeout: 4
+    *   Configure the source IP address of a selected interface for all outgoing TACACS+ packets as the loopback interface.
+*   Verification:
+    *   Confirm the successful creation of the Tacacs server with the specified parameters.
+
+### SYS-3.1.3 AAA Authentication Configuration Test
+
+*   Configuration:
+    *   Configure the DUT to use TACACS+ as the primary authentication method and local authentication as a fallback option.
+*   Verification:
+    *   Validate that the authentication settings are implemented as intended.
+
+### SYS-3.1.4 AAA Authorization Configuration Test
+
+*   Configuration:
+    *   Configure command authorization to exclusively utilize the TACACS+ server.
+    *   Configure authorization for configuration mode to primarily use the TACACS+ server and fall back to local authorization if the TACACS+ server is unavailable or does not respond.
+*   Verification:
+    *   Validate that the authorization settings are implemented and working as intended.
```

Comment:
Explicitly define the steps.  Ie:

1. Validate a user which is only defined in tacacs can authenticate using gnmi.
2. Block access to tacacs by configuring an ACL on the DUT and verify the local admin user authentication defined in SYS-3.1.1 succeeds using gnmi.

Reply:
The purpose of this test is solely to confirm proper configuration deployment via OC.  TACACS+ functionality is not evaluated here